flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -42,6 +42,26 @@ constructFlinkClassPath() {
     echo "$FLINK_CLASSPATH""$FLINK_DIST"
 }
 
+findFlinkDistJar() {
+    local FLINK_DIST
+
+    while read -d '' -r jarfile ; do
```

Comment:
Wouldn't it be easier to run one of the following?
```
$ ls lib/flink-dist_*.jar
lib/flink-dist_2.11-1.11-SNAPSHOT.jar
```
or
```
$ find lib -name 'flink-dist*.jar'
lib/flink-dist_2.11-1.11-SNAPSHOT.jar
```

If I understand correctly `FLINK_DIST` should contain only one jar.